sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -781,6 +783,16 @@ def run_ParDo(self, transform_node, options):
 
     step.add_property(PropertyNames.OUTPUT_INFO, outputs)
 
+    # Add the restriction encoding if we are a splittable DoFn
+    # and are using the Fn API on the unified worker.
+    from apache_beam.runners.common import DoFnSignature
+    signature = DoFnSignature(transform_node.transform.fn)
+    if (use_fnapi and use_unified_worker and signature.is_splittable_dofn()):
+      restriction_coder = (
+          signature.get_restriction_provider().restriction_coder())
+      step.add_property('restriction_encoding',
```

Comment:
Could you put this constant in PropertyNames?

Reply:
Done